parse_all.py

```diff
@@ -1,18 +1,48 @@
 import sys
+import argparse
+import os
 import json
 import importlib
+import shutil
+from paths import BASE_PATH, SOURCES_FILE, JSON_DIR, TMP_CASES, TMP_POPULATION
 
-SOURCES = "./sources.json"
+APP_PATH = 'src/assets/data'
 
 if __name__ == "__main__":
-    # initialize empty json
-    with open('case-counts/case_counts.json', 'w') as fh:
-        fh.write("{}")
-
-    srcs = list(json.load(open(SOURCES)).keys())
-    for src in srcs:
-        # Allow running this as `python parse_all.py netherlands` to filter sources (debug mode)
-        if len(sys.argv) == 1 or sys.argv[1] == src:
-            print(f"Running {src}", file=sys.stderr)
-            country = importlib.import_module(f"parsers.{src}")
-            country.parse()
+
+    parser = argparse.ArgumentParser(description = "",
+                                     usage="Parse data and copy output to app")
+
+    parser.add_argument('--fetch', action='store_true', help='update sources from remote')
+    parser.add_argument('--deploy', action='store_true', help='copy to app')
+    parser.add_argument('--parsers', nargs='+', help='parsers to run')
+    args = parser.parse_args()
+
+    if args.fetch:
+        # make directory for JSONs if it doesn't exist
+        outpath = os.path.join(BASE_PATH, JSON_DIR)
+        if not os.path.isdir(outpath):
+            os.mkdir(outpath)
+
+        # initialize empty json
+        with open(os.path.join(BASE_PATH, JSON_DIR, TMP_CASES), 'w') as fh:
+            fh.write("{}")
+
+        srcs = list(json.load(open(os.path.join(BASE_PATH, SOURCES_FILE))).keys())
+        for src in srcs:
+            # Allow running this as `python parse_all.py netherlands` to filter sources (debug mode)
+            if (args.parsers is None) or src in args.parsers:
+                print(f"Running {src}", file=sys.stderr)
+                country = importlib.import_module(f"parsers.{src}")
+                country.parse()
+
+        pop = importlib.import_module(f"scripts.populations")
+        pop.parse()
+
+    # copy jsons to app if requested
+    if args.deploy:
+        shutil.copy(os.path.join(BASE_PATH, JSON_DIR, TMP_CASES),
```

Comment:
Maybe some user feedback f"Deploying to { os.path.join(APP_PATH, 'case_counts.json')}" or so